doc/Language/control.pod6

```diff
@@ -535,6 +535,27 @@ Assigning C<Empty> will effectively undefine an Array, using C<for> over an
 undefined array will not even enter the loop, as shown, effectively behaving in
 the same way as above when C<Empty> was used directly.
 
+
+With C<hyper> and C<race> the for loop is potentially iterated in parallel.
+See also the documentaion for C<hyper> and C<race> in class L<Map|/type/Map>.
+
+
+=for code
+my $primes_h = hyper for ^10_000 -> $number { $number if $number.is-prime };
+say $primes_h.elems;
+say $primes_h.tail: 5; # OUTPUT: (9931 9941 9949 9967 9973)
```

Comment:
Please, use our unified format for output.

Reply:
See last section of https://github.com/perl6/doc/blob/master/CONTRIBUTING.md#documenting-types part.